docs/dev/auth.rst

```diff
@@ -44,6 +47,9 @@ The following additional configuration entries exist for auth in OS2MO.
 * ``"SAML_USERNAME_ATTR"``: If username is not read from NameID, the username
   will be read from an attribute with this name.
 
+All configuration of OS2MO is done as described in :ref:`settings`. The
+relevant section for authentication is called ``[saml_sso]``.
+
 Minimal example
```

Comment:
Please update this section with new config values.

Reply:
done